python/pyarrow/tests/test_adhoc_memory_leak.py

```diff
@@ -28,6 +31,7 @@
     pass
 
 
+@pytest.mark.numpy
```

Comment:
if something is already labeled as requiring pandas, it shouldn't need to be labeled as also requiring numpy?